website/src/writing-php-code/phpdoc-types.md

```diff
@@ -56,6 +56,13 @@ General arrays
 * `non-empty-array<Type>`
 * `non-empty-array<int, Type>`
 
+
+Array keys or values
```

Comment:
It's not just for arrays, but all iterables. Also an example with some code would be nice.

Reply:
Done